tools/sgartifactregistry/auth.go

```diff
@@ -24,13 +25,22 @@ func NpmAuthenticate(ctx context.Context, packageJSONDir, registryURL string) er
 	if err := cmd.Run(); err != nil {
 		return err
 	}
+
 	registry := strings.TrimPrefix(registryURL, "https://")
 	// Trailing slashes at the end of the URL have been known to cause issues with some setups
 	registry = strings.TrimSuffix(registry, "/")
 
+	expired, err := isGoogleAuthExpired(ctx)
+	if err != nil {
+		return fmt.Errorf("unable to verify Google authentication token expiration: %v", err)
+	}
+	if expired {
+		return fmt.Errorf("google authentication token is expired. Please authenticate using 'gcloud'")
```

Comment:
Should we add what exact commands to use to authenticate? Just to make it a little bit easier for someone that doesn't know the recommended way of authenticating?

Reply:
Yes. Can fix